src/app/api/upload-to-cdn/route.ts

```diff
@@ -0,0 +1,48 @@
+import { NextResponse } from 'next/server';
+import { v4 as uuidv4 } from 'uuid';
+
+const CDN_BASE_URL = `https://sg.storage.bunnycdn.com/${process.env.CDN_SZ_NAME}${process.env.CDN_BASE_PATH}`;
```

Comment:
The current setup has the  domain hardcoded in the `CDN_BASE_URL=https://sg.storage.bunnycdn.com`, which could cause confusion for contributors who don't select Singapore as their region.

There are two potential solutions to address this:

1. **Make the base URL configurable** by storing the full URL as an environment variable. This will allow contributors to easily set their region without modifying the code.
   
2. **Enforce the use of the Singapore zone** for all contributors. This ensures consistency and prevents issues, as the default region (Falkenstein) would lead to a 401 error.

![image](https://github.com/user-attachments/assets/cb02e4ba-6ebd-4661-9dcb-6ae7093a91b3)

By default, BunnyCDN auto-selects Falkenstein, DE (`storage.bunnycdn.com`), which can result in authentication errors. The correct mapping for regions is as follows:
- **Falkenstein, DE**: `storage.bunnycdn.com`
- **London, UK**: `uk.storage.bunnycdn.com`
- **New York, US**: `ny.storage.bunnycdn.com`
- **Los Angeles, US**: `la.storage.bunnycdn.com`
- **Singapore, SG**: `sg.storage.bunnycdn.com`
- **Stockholm, SE**: `se.storage.bunnycdn.com`
- **São Paulo, BR**: `br.storage.bunnycdn.com`
- **Johannesburg, SA**: `jh.storage.bunnycdn.com`
- **Sydney, SYD**: `syd.storage.bunnycdn.com`